src/other/Table.js

```diff
@@ -307,27 +325,40 @@
             .append("td")
         ;
         cells[this.renderHtmlDataCells() ? "html" : "text"](function (d, idx) { 
-            var retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            var retVal;
+            if (!(d instanceof Widget)) {
+                retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            }
             return retVal; 
         });
         cells.exit()
             .remove()
-        ;      
+        ;
+
         rows.each(function(tr,trIdx){
-            d3.select(this).selectAll("td").each(function(tdContents,tdIdx){
+            d3.select(this).selectAll("td").each(function(tdContents,tdIdx){    
```

Comment:
@dtsnell4 This selector will grab unexpected elements when a Table widget exists within another Table widget.